File: CS/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RichEditFieldsToValues
{
    public class Employee
    {
        string name;
        public string Name
        {
            get { return name; }
            set { name = value; }
        }
        string address;

        public string Address
        {
            get { return address; }
            set { address = value; }
        }

        public Employee(string name, string address)
        {
            this.name = name;
            this.address = address;
        }
    }
}

File: VB/Employee.vb
Public Class Employee

    Private nameField As String
    Public Property Name As String
        Get
            Return nameField
        End Get

        Set(ByVal value As String)
            nameField = value
        End Set
    End Property
    Private addressField As String
    Public Property Address As String
        Get
            Return addressField
        End Get

        Set(ByVal value As String)
            addressField = value
        End Set
    End Property
    Public Sub New(ByVal name As String, ByVal address As String)
        nameField = name
        addressField = address
    End Sub
End Class
